infra/README.md

```diff
@@ -0,0 +1,44 @@
+Preqequisites:
```

Comment:
Currently, across the codebase we've generally been preferring RST to MD for text files.  It seems a bit more full featured, and more importantly it's the same syntax that `Sphinx` uses, which we use for our documentation in general.  Do you have opinions about RST vs MD?

Reply:
I'll change it, I don't have a preference between the two